packages/xod-arduino/platform/program.tpl.cpp

```diff
@@ -45,6 +45,41 @@ void runTransaction() {
     XOD_TRACE_F("Transaction started, t=");
     XOD_TRACE_LN(g_transactionTime);
 
+    #if defined(XOD_DEBUG) || defined(XOD_SIMULATION)
```

Comment:
```suggestion
#if defined(XOD_DEBUG) || defined(XOD_SIMULATION)
```

Do not indent preprocessor directives. This is legacy, mainly, but so strong that some editors bark on you if it is indented.